unit_tests/testDBC/testDBC.f90

```diff
@@ -0,0 +1,81 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+PROGRAM testDBC
+#include "DBC.h"
+  USE DBC
+  USE IntrType
+  IMPLICIT NONE
+  CHARACTER(LEN=*),PARAMETER :: modName="testDBC"
+  INTEGER(SIK) :: iopt
+  CHARACTER(LEN=5) :: arg
+#ifdef HAVE_MPI
+  INCLUDE "mpif.h"
+  INTEGER(SIK) :: mpierr
+  CALL MPI_Init(mpierr)
+#endif
+
+  CALL GET_COMMAND_ARGUMENT(1,arg)
+  READ(arg,*) iopt
+
+  SELECTCASE(iopt)
+    CASE(1)
+      CALL require_pass()
+    CASE(2)
+      CALL ensure_pass()
+    CASE(3)
+      CALL require_fail()
+    CASE(4)
+      CALL ensure_fail()
+    CASE DEFAULT
+      STOP -1
+  ENDSELECT
+
+#ifdef HAVE_MPI
+      CALL MPI_Finalize(mpierr)
+#endif
+!
+!===============================================================================
+  CONTAINS
+!
+!-------------------------------------------------------------------------------
+!> @brief Tests the REQUIRE macro and associated function calls
+!>
+    SUBROUTINE require_pass()
+      WRITE(*,*) "Testing REQUIRE Passing"
+      DBC_REQUIRE(5==5)
+
+    ENDSUBROUTINE require_pass
+!
+!-------------------------------------------------------------------------------
+!> @brief Tests the ENSURE macro and associated function calls
```

Comment:
Commenting unit tests!? What wizardry is this?

Reply:
I can delete if desired?? ;-)